### PR TITLE
Game and author

### PR DESCRIPTION
**Pull request by Uzair-Manzoor**
## The following is a list of the accomplished tasks while working on this part of the project:

- [x] Created a feature branch for this milestone.
- [x] Created the Game and Author classes.
- [x] Implemented Game and Author properties.
- [x] Implemented Methods in the Game and Author classes.
- [x] Present the user with a list of options to add a new game, list authors, and list games.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,8 +1,73 @@
...
+  puts "\nOptions:"
   puts '1. List of games'
-  puts '2. List all genres'
-  puts '3. List all labels'
-  puts '4. Add a game'
-  puts '5. Quit'
+  puts '2. List all authors'
+  puts '3. Add a game'
+  puts '4. Quit'
+end
```
Comment (ismayito) [inline main.rb:56]: Thanks for this but I believe with time we shall have to refactor this code of all methods such that we have them in separate files to increase maintainability and readability of the code.

File: main.rb [hunk 2]
```diff
@@ -1,8 +1,73 @@
...
+  first_name = gets.chomp
+
+  puts 'Enter the Author Last Name:'
+  last_name = gets.chomp
+
+  author = Author.new(items.size + 1, first_name, last_name)
+  new_game = Game.new(title, publish_date, multiplayer, last_played_at, author)
+
+  add_item(new_game, items)
+end
```
Comment (ismayito) [inline main.rb:46]: Thanks for this but I believe with time we shall have to refactor this code of all methods such that we have them in separate files to increase maintainability and readability of the code.

File: main.rb [hunk 3]
```diff
@@ -1,8 +1,73 @@
...
+  games.each_with_index { |game, index| puts "Game ID: #{index + 1}, Title: #{game.title}" }
+end
+
+def add_game(items)
+  puts 'Enter Game Title:'
+  title = gets.chomp
+
+  puts 'Is it Multiplayer? (true/false):'
+  multiplayer = gets.chomp == 'true'
+
```
Comment (ismayito) [inline main.rb:29]: Thanks for this but I believe with time we shall have to refactor this code of all methods such that we have them in separate files to increase maintainability and readability of the code.

**Review by ismayito — APPROVED** [review]
Hello @Uzair-Manzoor . Thanks for this, You have followed the correct git flow, linters are passing, correct description.
You can now merge your pull request and move forward.

File: main.rb (hunk 3 above)
Comment (Uzair-Manzoor) [reply main.rb:29]: Hi @ismayito thank you for the insightful feedback. Changes will be made accordingly in the upcoming tasks.
Cheers

**Pull request merged**